libindi/drivers/telescope/celestrondriver.cpp

```diff
@@ -185,15 +185,19 @@ int CelestronDriver::send_command(const char *cmd, int cmd_len, char *resp,
         }
     }
 
-    if (resp_len == 0)
-        return true;
-
     if (nbytes != resp_len)
     {
-        LOGF_ERROR("Received %d bytes, expected %d.", nbytes, resp_len);
+        hex_dump(hexbuf, resp, nbytes > resp_len ? nbytes : resp_len);
+        LOGF_ERROR("Received %d bytes, expected %d <%s>", nbytes, resp_len, hexbuf);
         return 0;
     }
 
+    if (resp_len == 0)
+    {
+        LOG_DEBUG("resp_len 0, no response expected");
```

Comment:
wouldn't this spam the debug log? many commands have resp_len = 0

Reply:
I don't think so, the commands with no reply still return a # character.